engine/engine/tests/test_views.py

```diff
@@ -39,4 +27,4 @@ def test_startupprobe_populates_integrations_cache(settings, detached_integratio
         response = client.get("/startupprobe/")
 
     assert response.status_code == status.HTTP_200_OK
-    assert mock_update_cache.called == is_cache_updated
+    mock_update_cache.assert_called_once
```

Comment:
I believe `assert_called_once` needs to be invoked as it is a method ([docs](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once))